### ISSUE TITLE
Any plans for mobile apps?

### ISSUE THREAD

**Issue by loganto** [Sonerezh/sonerezh#217]
Project looks very promising.  
But how would I use it from Android for example?   
Is the idea that people would use via mobile browser?

**Comment by MightyCreak** [comment]
I was able to use Sonerezh simply with Firefox, but I agree that a dedicated app might be a good idea. I'm not one of the dev though ;)

**Comment by kiwiASRALL** [comment]
It would be really great! with local cache to listen music offline. Using firefox works but it is not comfortable

**Comment by peterbrinck** [comment]
I believe they've mentioned that it's on their todo, but I'm not 100% sure

**Comment by MightyCreak** [comment]
It's a work in progress: https://roadmap.sonerezh.bzh/b/eCuz4jyhwjmxpxX8C/sonerezh-roadmap/i5ACxELvqqnYQhT8m

**Comment by kiwiASRALL** [comment]
1 year..

**Comment by MightyCreak** [comment]
Well, they're doing this project on their spare time. I would also like to see more features coming in, but projects done on spare time have significantly slower pace than other open source projects sponsored by companies, like ownCloud for instance.

**Comment by lGuillaume124** [comment]
> Well, they're doing this project on their spare time. I would also like to see more features coming in, but projects done on spare time have significantly slower pace than other open source projects sponsored by companies, like ownCloud for instance.

Indeed.

**Comment by TheUltimateC0der** [comment]
Is there any progress on this ?

**Comment by lGuillaume124** [comment]
Nope. I suppose Sonerezh should have its own API before starting to think about a mobile app.
